rest-api/test/unit_test/api_test/biobank_order_api_test.py

```diff
@@ -374,6 +375,16 @@ def test_insert_new_order(self):
     _strip_fields(full_order_json)
     self.assertEquals(full_order_json, result)
 
+  def test_biobank_history_on_insert(self):
+    with self.bio_dao.session() as session:
+      self.summary_dao.insert(self.participant_summary(self.participant))
+      order_json = load_biobank_order_json(self.participant.participantId,
+                                           filename='biobank_order_2.json')
+      result = self.send_post(self.path, order_json)
+      load_biobank_order_json(self.participant.participantId, filename='biobank_order_1.json')
+      order_history = session.query(BiobankOrderHistory).first()
+      self.assertEqual(result['id'], order_history.biobankOrderId)
```

Comment:
Also check version?

Can we also check to make sure the identifiers and samples got inserted into their history tables?

Check that this history record sticks around after an update?

Reply:
done